plugins/source/aws/docs/tables/aws_resiliencehub_app_assessments.md

```diff
@@ -24,13 +24,13 @@ The following tables depend on aws_resiliencehub_app_assessments:
 |_cq_sync_time|Timestamp|
 |_cq_id|UUID|
 |_cq_parent_id|UUID|
-|account_id|String|
-|region|String|
+|account_id (PK)|String|
+|region (PK)|String|
+|app_arn (PK)|String|
 |arn (PK)|String|
```

Comment:
Is the `account_id` and `region` not in the ARN?

Reply:
I decided to make the PKs consistent for this service

Reply:
In the rest of the AWS plugin we try to use as few PK fields as possible... Why is diverging from the rest of the plugin worth it here?

Reply:
https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsresiliencehubservice.html#awsresiliencehubservice-resources-for-iam-policies
I'll update the code accordingly